### DISCUSSION TITLE
the POST requests to the API (api/v2/services/dhcp_server/static_mapping) gradually become slower to execute

### DISCUSSION THREAD

**Discussion by philippenc** [jaredhendrickson13/pfsense-api#532] [Q&A]
Hello,

We are currently using interfaces with the DHCP server, only enabling static mapping. We have noticed that when the number of static mappings exceeds a hundred on a DHCP server, the POST requests to the API (api/v2/services/dhcp_server/static_mapping) gradually become slower to execute.

For instance, although my Flask server is configured with the options gunicorn --timeout 0 --bind 127.0.0.1:5000 wsgi:app, it encounters timeout issues because the response to the request is not received in time. The addition of a static IP address is only visible on pfSense one minute after the request is made.

Could you assist us in resolving this latency issue?

Best regards,

**Comment by jaredhendrickson13** [comment]
I can reproduce and I've just opened issue #537 for this. Feel free to follow that issue for updates.
